File: hacker_rank/30_days_of_code/day_5.py
'''
Objective
In this challenge, we will use loops to do some math. Check out the Tutorial tab to learn more.

Task
Given an integer, , print its first  multiples. Each multiple  (where ) should be printed on a new line in the form: n x i = result.

Example

The printout should look like this:

https://www.hackerrank.com/challenges/30-loops
'''
#!/bin/python3

import math
import os
import random
import re
import sys



if __name__ == '__main__':
    n = int(input().strip())
    for i in range(1,11):
        ans = n * i
        print(n,"x",i,"=",ans)

File: hacker_rank/30_days_of_code/cpp/day_5.cpp
//https://www.hackerrank.com/challenges/30-loops

#include <iostream>

using namespace std;

int main() {
    int N;
    cin >> N;

    for (int i = 1; i < 11; i++) {
        printf("%i x %i = %i\n", N, i, N * i);
    }

    return 0;
}